src/index.js

```diff
@@ -0,0 +1,77 @@
+import './styles.css';
+import { checkEvent, currentChecks } from './storage';
+import { setEventToLocalStorage, getEventFromLocalStorage } from './functional';
+
+// Array Of Event Tasks To Do
+const eventList = [
+  {
+    description: 'Play video games',
+    completed: false,
+    index: 0,
+  },
+  {
+    description: 'Go Shopping',
+    completed: true,
+    index: 1,
+  },
+  {
+    description: 'Cook Dinner',
+    completed: false,
+    index: 2,
+  },
+  {
+    description: 'Listen to music',
+    completed: true,
+    index: 3,
+  },
+  {
+    description: 'Read Novels',
+    completed: false,
+    index: 4,
+  },
```

Comment:
- [ ] On page load, the eventlist array is should be rendered dynamically to the DOM.